backend/src/api/message.py

```diff
@@ -21,17 +22,14 @@ def get_usecase(injector=Depends(get_injector)) -> CreateMessageUseCaseIf:
     return injector.get(CreateMessageUseCaseIf)
 
 
-def get_channel_access_checker(injector=Depends(get_injector)) -> ChannelAccessCheckerUseCaseIf:
-    return injector.get(ChannelAccessCheckerUseCaseIf)
-
-
 async def check_channel_access_for_message(
```

Comment:
関数名が冗長な気がするので、以下のような形で大丈夫かと思います。
```suggestion
async def check_channel_access(
```